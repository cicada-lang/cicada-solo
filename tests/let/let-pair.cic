x: (Nat) * Nat =
  one = add1(zero)
  two = add1(one)
  three = add1(two)
  cons(one, two)

x
