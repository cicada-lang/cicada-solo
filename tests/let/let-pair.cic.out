cons(1, 2): (_: Nat) * Nat
